### PR TITLE
Makefile Refactor (ctd)

### PR DESCRIPTION
**Pull request by matentzn**
This is a continuation in my quest to improve the `Makefile` pipeline (reduce runtime in particular, but also certain ERRORs and WARNINGs).

### CODE REVIEW

File: src/ontology/mondo-ingest.Makefile [hunk 1]
```diff
@@ -362,16 +356,20 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
-tmp/mondo.owl tmp/mondo.sssom.tsv:
+tmp/mondo_repo_built:
```
Comment (matentzn) [inline src/ontology/mondo-ingest.Makefile:359]: I created a separate goal for checking out the MONDO repo as the `make` pipeline did have some issues with the "multi-target" solution before (essentially rerunning the goal twice).

File: src/ontology/mondo-ingest.Makefile [hunk 2]
```diff
@@ -274,13 +269,12 @@ $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_i
 	--outpath $@
 
 # Exclusions: all artefacts for single ontology
-exclusions-%:
-	$(MAKE) $(REPORTDIR)/$*_term_exclusions.txt
-	$(MAKE) $(REPORTDIR)/$*_exclusion_reasons.ttl
-	$(MAKE) $(REPORTDIR)/$*_excluded_terms_in_mondo_xrefs.tsv
+.PHONY: exclusions-%
+exclusions-%: reports/%_exclusion_reasons.ttl reports/%_excluded_terms_in_mondo_xrefs.tsv  $(REPORTDIR)/%_term_exclusions.txt 
+	echo "$@ completed"
```
Comment (matentzn) [inline src/ontology/mondo-ingest.Makefile:274]: I undid a lot of the `$(MAKE)` pattern which I orginally introduced in good faith, not knowing that `-B` is passed into $(MAKE) as well.. which mean that a lot of goals were run multiple times for no reason.

File: src/ontology/mondo-ingest.Makefile [hunk 3]
```diff
@@ -489,22 +487,24 @@ slurp/%.tsv: $(COMPONENTSDIR)/%.owl $(TMPDIR)/mondo.sssom.tsv $(REPORTDIR)/%_ter
 	--outpath $@
 
 .PHONY: slurp-%
-slurp-%:
-	$(MAKE) slurp/$*.tsv -B
+slurp-%: slurp/%.tsv
+	echo "$@ completed".
```
Comment (matentzn) [inline src/ontology/mondo-ingest.Makefile:491]: For some reason, `make` wants to see a goal body (in this case the `echo`) to run. Else, without it, `make` claims that `slurp-omim` does not exist as a goal.

File: src/sparql/fix_xref_prefixes.ru [hunk 4]
```diff
@@ -2,22 +2,24 @@ prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
...
+
 
 DELETE {
   ?entity xref: ?value .
   ?ax a owl:Axiom ;
          owl:annotatedSource ?entity ;
          owl:annotatedProperty xref: ;
          owl:annotatedTarget ?value ;
-         ?p ?v .
+         <http://purl.obolibrary.org/obo/ECO_0000218> ?v .
```
Comment (matentzn) [inline src/sparql/fix_xref_prefixes.ru:14]: Simplifying this query as there is no other metadata we want to copy other than this one.

File: src/sparql/fix_xref_prefixes.ru [hunk 5]
```diff
@@ -27,8 +29,14 @@ WHERE
...
            owl:annotatedProperty xref: ;
            owl:annotatedTarget ?value ;
-           ?p ?v .
+           <http://purl.obolibrary.org/obo/ECO_0000218> ?v .
   }
   FILTER (isIRI(?entity))
-  BIND(REPLACE(REPLACE(REPLACE(REPLACE(?value, "ICD-11:", "ICD11:", "i"), "ICD-10:", "ICD10:", "i"), "MeSH:", "MESH:", "i"), "OMIM:PS", "OMIMPS:", "i") AS ?value_fixed)
+  BIND(
+    REPLACE(REPLACE(REPLACE(REPLACE(REPLACE(?value, 
+        "\u00A0", "", "i"),
```
Comment (matentzn) [inline src/sparql/fix_xref_prefixes.ru:37]: This is necessary to compensate for https://github.com/Orphanet/ORDO/issues/37

File: src/ontology/mondo-ingest.Makefile [hunk 6]
```diff
@@ -21,12 +21,7 @@ pip-%:
 	 python3 -m pip install --upgrade $*
 
 .PHONY: dependencies
-dependencies:
-	$(MAKE) pip-pip
-	$(MAKE) pip-setuptools
-	$(MAKE) pip-oaklib
-	$(MAKE) pip-sssom
-	$(MAKE) pip-semsql
+dependencies: pip-pip pip-setuptools pip-oaklib pip-sssom pip-semsql
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:24]: I suppose this change is effectively the same and more succinct. Ah, and I see that you're saying `-B` gets passed down.

File: src/ontology/mondo-ingest.Makefile (hunk 2 above)
Comment (joeflack4) [reply src/ontology/mondo-ingest.Makefile:274]: Oh, didn't realize that! I think I've seen that before. Good to know this.

File: src/ontology/mondo-ingest.Makefile (hunk 3 above)
Comment (joeflack4) [reply src/ontology/mondo-ingest.Makefile:491]: Really? Troubleshooting `make` can be frustrating. Glad you figured this out.

**Review by joeflack4 — APPROVED** [review]
Glad to have these changes!

What does "CTD" mean?

**Pull request merged**